README.md

```diff
@@ -42,10 +44,10 @@ The plugin folder contains a video "Example_BP_PubSub.mp4" demonstrating how to
 
 An example setup using this plugin can be found at [turtlebot3-UE](https://github.com/rapyuta-robotics/turtlebot3-UE)
 
-# Notes on working with ROS2 and UE4
+# Notes on working with ROS2 and UE
 
-- rcl and void\* types cannot be managed by UE4 (no UPROPERTY) and therefore can't be used directly in Blueprint. Whenever access to these variables is needed, the user should write a class to wrap it and all of their handling must be done in C++.
-- some basic numerical types are not natively supported in Blueprint (e.g. double, unsigned int). In order to use these, a workaround is needed (a plugin implementing those types for BP, a modified UE4 or a custom implementation).
+- rcl and void\* types cannot be managed by UE (no UPROPERTY) and therefore can't be used directly in Blueprint. Whenever access to these variables is needed, the user should write a class to wrap it and all of their handling must be done in C++.
+- some basic numerical types are not natively supported in Blueprint (e.g. double, unsigned int). In order to use these, a workaround is needed (a plugin implementing those types for BP, a modified UE or a custom implementation).
 - In autogenerated messages, the method MsgToString() should be implemented by the user as its current purpose is to help debugging.
 
 # How to update ROS inside RclUE
```

Comment:
... to automatically build and update ROS2 ... Follow steps 1) for building and copying ros2 base library under rclUE and 2) for generating UE src codes to access msg from UE.